game/game.py

```
@@ -79,123 +84,126 @@ def get_state(self, player_id):
             self.passed[self.turn] = True
             # 全員パス or 最後に出した人以外全員パス → 場リセット
-            others_passed = all(self.passed[i] or len(self.players[i].hand) == 0 for i in range(self.num_players) if i != self.turn)
-            if self.last_player == self.turn and others_passed:
-                self._reset_field()
-                reset_happened = True
-                self.turn = self.last_player
-                return self.get_state(self.turn), 0.0, False, reset_happened
-        else:
-            self.last_player = self.turn
+            if not empty_field:
+                if self.last_player == self.turn and self._all_others_passed():
```

Ruff (0.12.2)

Use a single `if` statement instead of nested `if` statements

(SIM102)